tests/core/document/test_partial_document_update.py

```diff
@@ -201,8 +202,9 @@ def test_update_text_field_tensor(self):
             "text_field_tensor": "I can't be updated",
             "_id": "1"
         }
-        r = update_documents(body=UpdateDocumentsBodyParams(documents=[updated_doc]),
-                             index_name=self.structured_index_name, marqo_config=self.config)
+        r = self.config.document.partial_update_documents_by_index_name(
```

Comment:
I see we changed from testing the api method to 'service' method. I reckon the intention is to test the 'right' thing?
In this case, should we not include `dict(exclude_none=True, by_alias=True)` but to check the pydantic model directly? Then we might need another test to test the `.dict(exclude_none=True, by_alias=True)` part.

Reply:
Your understanding is correct. The only reason I am not doing this is that would take much more effort to revise the unit tests code. I am not sure if we have enough time to do this in this release